index.html

```diff
@@ -8,31 +8,56 @@
     <link rel="stylesheet" type="text/css" href="css/grid.css">
     <!-- Add a link to your custom CSS here -->
     <link rel="stylesheet" type="text/css" href="css/main.css">
-
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
     <title>Responsive grid project</title>
 </head>
-
+<header class="header-wrapper col-12 sm-col-12 md-col-12 lg-col-12 xl-col-12">
+  <section class="header-logo ">
+    <img class="logo col-6 sm-col-6 md-col-4 lg-col-4 xl-col-2" src="css/1280px-NHS-Logo.svg.png" alt="Company logo"/>
+  </section>
+  <nav class="header-nav col-6 sm-col-4 md-col-4 lg-col-3 xl-col-3 ">
+    <ul class="col-6 sm-col-8 md-col-4 lg-co3 xl-col-3 ">
+      <li>Home</li>
+      <li>About</li>
+      <li>Contact</li>
+    </ul>
+  </nav>
+</header>
 <body>       
-  <header class="header-wrapper col-12 sm-col-12 md-col-12 lg-col-12 xl-col-12">
-    <section class="header-logo col-2 sm-col-2 md-col-2 lg-col-2 xl-col-2">
-      <img src="/logo.svg" alt="Company logo"/>
-    </section>
-    <nav class="header-nav row">
-      <ul class="col-6 sm-col-6 md-col-6 lg-col-6 xl-col-6 ">
-        <li>Home</li>
-        <li>About</li>
-        <li>Contact</li>
-      </ul>
-    </nav>
-  </header>
+
     <div class="supercontainer col-12">
-        <img src="css/Craig-Titus-720x520.jpg" alt="Titus" class="col-12 sm-col-12 md-col-12 lg-col-12 xl-col-12">
-        <div class="text-block col-2 sm-col-2 md-col-2 lg-col-2 xl-col-2">
-          <h4>Craig Titus</h4>
-          <p>Fall of the legend</p>
+        <img  src="css/Craig-Titus-720x520.jpg" alt="Titus" class="bigpic col-12 sm-col-12 md-col-12 lg-col-12 xl-col-12">
+        <div class="text-block col-6 sm-col-8 md-col-4 lg-co3 xl-col-3">
+          <h2>Quit smoking</h2>
+          <p>Before it's too late</p>
         </div>
       </div>
```

Comment:
Is this the div to close the super container, if so ensure it is aligned with the opening div